action.yml

```diff
@@ -138,8 +138,9 @@ runs:
               repo: context.repo.repo,
             });
 
+            directory = '${{ inputs.directory }}'.split(' ')[0]
             botComment = comments.find(comment => {
-              return comment.user.type === 'Bot' && comment.body.includes('Terraform check on')
+              return comment.user.type === 'Bot' && comment.body.includes('Terraform check on `${directory}`)
             });
```

Comment:
Since we accept multiple directory in the input, perhaps we should check for all of them instead of just the first one. Something like this:
```javascript
directories = '${{ inputs.directory }}'.split(' ')
botComment = comments.find(comment => {
  return comment.user.type === 'Bot' && directories.map(dir => comment.body.includes('Terraform check on `${dir}`')).every(Boolean)
});
```

Reply:
Yeah I think its a good idea to check all the directories, updated in fe21fa8329a424a0dee2b89f336bdf86468aa191